features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/semantics/RootSemanticsNodeMapper.kt

```diff
@@ -116,7 +137,32 @@ internal class RootSemanticsNodeMapper(
         return semanticsNode.config.contains(SemanticsActions.SetText)
     }
 
+    @UiThread
+    private fun updateTouchOverrideAreas(
+        semanticsNode: SemanticsNode,
+        touchPrivacyManager: TouchPrivacyManager
+    ) {
+        val touchPrivacy = semanticsUtils.getTouchPrivacyOverride(semanticsNode)
+
+        if (touchPrivacy != null) {
+            val viewArea = semanticsNode.boundsInRoot.toAndroidRectF().toRect()
+            try {
+                val privacyLevel =
+                    TouchPrivacy.valueOf(touchPrivacy.toString().uppercase(Locale.US))
+                touchPrivacyManager.addTouchOverrideArea(viewArea, privacyLevel)
+            } catch (e: IllegalArgumentException) {
+                (Datadog.getInstance() as? FeatureSdkCore)?.internalLogger?.log(
+                    InternalLogger.Level.ERROR,
+                    listOf(InternalLogger.Target.USER, InternalLogger.Target.TELEMETRY),
+                    { INVALID_PRIVACY_LEVEL_ERROR },
+                    e
+                )
+            }
```

Comment:
minor suggestion: it is possible to avoid exception. Just iterate over `TouchPrivacy.values()` and then `firstOrNull`

Reply:
Actually we don't even have to iterate, `touchPrivacy` can be directly used, udpated